brevo.com.domain-authentication.json

```diff
@@ -3,31 +3,31 @@
   "providerName": "Brevo",
   "serviceId": "domain-authentication",
   "serviceName": "Brevo Domain Authentication",
-  "version": 3,
+  "version": 4,
   "syncPubKeyDomain": "brevo.com",
   "syncRedirectDomain": "app.brevo.com",
   "logoUrl": "https://static.brevo.com/meetings/assets/brevo_logo.png",
   "description": "Places TXT record for domain verification and DKIM records to authenticate email sent by Brevo on behalf of the user",
-  "variableDescription": "The variable %verfication_code% represents a combination of prefix string and a MD5 hash. %dkim_selector% is the DKIM selector for placing the DKIM public key. This needs to be a variable because the selector is dynamically generated. %dkim_public_key% is the DKIM public key. %dmarc_value% is dmarc value. %dkimcname% is cname value where DKIM will point",
+  "variableDescription": "The variable %verfication_code% represents a combination of prefix string and a MD5 hash. %dkim_host% is the host where DKIM key will be placed. %code_host% is the host where %verfication_code% will be placed. %dmarc_host% is the host where DMARC record will be placed. %dkim_public_key% is the DKIM public key. %dmarc_value% is dmarc value. %dkimcname% is cname value where DKIM will point",
   "records": [
     {
       "groupId": "code",
       "type": "TXT",
-      "host": "@",
+      "host": "%code_host%",
       "data": "%verfication_code%",
       "ttl": 3600
     },
     {
       "groupId": "dkim_raw",
       "type": "TXT",
-      "host": "%dkim_selector%._domainkey",
+      "host": "%dkim_host%",
       "data": "%dkim_public_key%",
       "ttl": 3600
     },
     {
       "groupId": "dmarc",
       "type": "TXT",
-      "host": "_dmarc",
+      "host": "%dmarc_host%",
```

Comment:
Just so I get the changes right... why would _dmarc ever be on a different host?

Reply:
We have a use case where we would add dmarc to subdomain.
For example: `_dmarc.abcd` 
In current template we would not able to add dmarc record on subdomain

Reply:
Unterstood. Would in this case all the other records land on the same subdomain?

Reply:
Yes all record will be on same subdomain, just difference of host.
For example dkim can be `mail._domainkey.abcd` and dmarc will be `_dmarc.abcd`

Reply:
I guess then this template change is not needed at all? Just use `host` parameter when applying the template.

Test yourself here: https://pdnsadmin.revproxy.short-lived.de/dc/free/templateedit

My test:
<img width="1210" alt="image" src="https://github.com/Domain-Connect/Templates/assets/28707030/1f87acf0-3e7b-4f8f-88f7-bf83945828ec">

Reply:
Thanks a lot @pawel-kow 
That's exactly I wanted it to work. Now No further need to change template.
We can close this PR